migrations/v6-MIGRATION.md

```diff
@@ -74,6 +74,15 @@ StoreProduct has been made an interface, which `GoogleStoreProduct` and `AmazonS
 
 ### Purchasing APIs
 
+The `purchasePackage()` and `purchaseProduct()` APIs have a new behavior for selecting which offer is used when
```

Comment:
nit and also take it or leave it: we could say "instead of choosing the base plan subscriptionoption" or something, to be clear what it's changing from